game/cards/dm06/cyber_lord.go

```diff
@@ -30,3 +30,42 @@ func Sopian(c *match.Card) {
 
 	c.Use(fx.Creature, fx.TapAbility)
 }
+
+func Kyuroro(c *match.Card) {
+
+	c.Name = "Kyuroro"
+	c.Power = 2000
+	c.Civ = civ.Water
+	c.Family = []string{family.CyberLord}
+	c.ManaCost = 6
+	c.ManaRequirement = []string{civ.Water}
+
+	c.Use(fx.Creature, fx.When(fx.BreakShield, func(card *match.Card, ctx *match.Context) {
+
+		event, _ := ctx.Event.(*match.BreakShieldEvent)
```

Comment:
I think we should check for `ok` and `return` here if the event is of another type even though it's already being checked above in `fx.breakShield` just in case someone makes changes to `fx.BreakShield` in the future and don't know that it will cause this to panic

Reply:
ok, fixed.